cli/pkg/kctrl/cmd/core/examples.go

```diff
@@ -14,6 +14,13 @@ type PackageCommandTreeOpts struct {
 
 	Color bool
 	JSON  bool
+
+	DefaultKubeconfigOverridePath    string
+	DefaultKubeconfigOverrideContext string
+
+	DefaultServiceAcccountName string
+	UnsetWaitByDefault         bool
```

Comment:
I actually liked the previous name for this flag more, then `Unset` part in `UnsetWaitByDefault` sounds a little bit redundant.

Reply:
Trying to keep the language right 🤔 
I could flip the logic and make it `WaitByDefault`?

Reply:
Yeah, that sounds more intuitive.

Reply:
My intention was to keep the changes in the other files to a minimum. Flipping the logic would mean we touch more files 🤔